tests/screenshot/apps/components/SwitchItem.js

```diff
@@ -40,6 +40,7 @@ const SwitchItemTests = [
 		<SwitchItem>Focused SwitchItem</SwitchItem>,
```

Comment:
Do you think we need // [QWTC-2130] above line 40 for step 4?

Reply:
You are correct. I updated comment for QWTC-2130.